docs/tutorial/cspc/allow-tagged-bds/allowed-bds.md

```diff
@@ -0,0 +1,144 @@
+# Block Device Tagging
+
+Node disk manager provides a feature to restrict the use of block devices via
+tagging it. Engines like cStor utilises this feature to apply restriction on 
+blockdevices and ensure that only cStor can use it and hence guarantee that the 
```

Comment:
NDM provides a feature to reserve block devices to be used for specific applications via block device tags. This feature can also be used by cStor operators to specify the block devices that should be consumed by cstor pools and conversely restrict anyone else from using those block devices. This feature can help in protecting against manual errors in specifying the block devices in the CSPC yamls by users.